docs/dev-guide/iframe.md

```diff
@@ -2,47 +2,101 @@
+
+You can find all you need to get started and managing your Jitsi implementation on this page. It includes the following topics:
+
+* [Installation](#installation)
+
+* [Creating the Jitsi Meet API object](#creating-the-jitsi-meet-api-object)
+
+* [Configuring the tile view](#configuring-the-tile-view)
+
+* [Controlling the embedded Jitsi Meet Conference](#controlling-the-embedded-jitsi-meet-conference)
+
+* [Commands](#commands)
+
+* [Multiple command execution](#multiple-command-execution)
+
+* [Event listeners](#event-listeners)
+
+## **Installation**
+
+To enable the Jitsi Meet API in your application you must use one of the following JavaScript (JS) Jitsi Meet API library scripts and integrate it into your appication:
+
+For self-hosting in your domain:
 
-Self-hosted:
 ```javascript
 <script src='https://<your-domain>/external_api.js'></script>
 ```
 
 meet.jit.si:
 ```javascript
 <script src='https://meet.jit.si/external_api.js'></script>
-```
-
-## API
-
-### `api = new JitsiMeetExternalAPI(domain, options)`
 
-The next step for embedding Jitsi Meet is to create the Jitsi Meet API object.
-Its constructor takes a number of options:
-
-* **domain**: domain used to build the conference URL, 'meet.jit.si' for
-  example.
-* **options**: object with properties - the optional arguments:
-    * **roomName**: (optional) name of the room to join.
-    * **width**: (optional) width for the iframe which will be created. If a number is specified it's treated as pixel units. If a string is specified the format is number followed by 'px', 'em', 'pt' or '%'.
-    * **height**: (optional) height for the iframe which will be created. If a number is specified it's treated as pixel units. If a string is specified the format is number followed by 'px', 'em', 'pt' or '%'.
-    * **parentNode**: (optional) HTML DOM Element where the iframe will be added as a child.
-    * **configOverwrite**: (optional) JS object with overrides for options defined in [config.js].
-    * **interfaceConfigOverwrite**: (optional) JS object with overrides for options defined in [interface_config.js].
-    * **noSSL**: (optional, defaults to false) Boolean indicating if the server should be contacted using HTTP or HTTPS.
-    * **jwt**: (optional) [JWT](https://jwt.io/) token.
-    * **onload**: (optional) handler for the iframe onload event.
-    * **invitees**: (optional) Array of objects containing information about new participants that will be invited in the call.
-    * **devices**: (optional) A map containing information about the initial devices that will be used in the call.
-    * **userInfo**: (optional) JS object containing information about the participant opening the meeting, such as `email`.
+```
 
-Example:
+## **Creating the Jitsi Meet API object**
+
+After you have installed the Meet API library, you must then create the Jitsi Meet API object.
```

Comment:
There is really no installation process.

Reply:
Changed title to Activation